cmd/reconAccuracy/reconAccuracy.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/fasta"
+	"log"
+)
+
+//TODO: this code will need to change drastically for sequences of varying lengths.
+//The loop through the sequence is restricted by a single fasta and the tot calculation will need to calculate the total number of bps
+//ReconAccuracy calculates the total number of incorrectly reconstructed base pairs in a tree and returns a percentage of correct base calls
+func ReconAccuracy(simFilename string, reconFilename string) {
+	var allNodes string
+	allNodes = "all Nodes"
+	var found bool = false
+	tot := 0.0
+	sim := fasta.Read(simFilename)
+	recon := fasta.Read(reconFilename)
+
+	answer := make(map[string]float64)
+
+	for i := 0; i < len(sim); i++ {
+		num := 0.0
```

Comment:
I would declare "num" once at the top of the function "var num float64" and then just initialize it to zero here "num = 0".  This way it keeps being declared over and over (in theory making this faster although probably not noticable) and it can also lead to some tricky bugs if there is another "num" in a different scope (these can be a nightmare to find), or if you try to access this after the loop.